addons/l10n_it_edi/models/account_move.py

```diff
@@ -125,6 +125,11 @@ class AccountMove(models.Model):
         copy=False,
     )
 
+    l10n_it_amount_vat_signed = fields.Monetary(string='VAT', compute='_compute_amount_extended', currency_field='company_currency_id')
+    l10n_it_amount_pension_fund_signed = fields.Monetary(string='Pension Fund', compute='_compute_amount_extended', currency_field='company_currency_id')
+    l10n_it_amount_withholding_signed = fields.Monetary(string='Withholding', compute='_compute_amount_extended', currency_field='company_currency_id')
+    l10n_it_amount_before_withholding_signed = fields.Monetary(string='Total Before Withholding', compute='_compute_amount_extended', currency_field='company_currency_id')
+
```

Comment:
Please adapt to the style of the fields around these
```suggestion
    l10n_it_amount_vat_signed = fields.Monetary(
        string='VAT',
        compute='_compute_amount_extended',
        currency_field='company_currency_id'
    )
    l10n_it_amount_pension_fund_signed = fields.Monetary(
        string='Pension Fund',
        compute='_compute_amount_extended',
        currency_field='company_currency_id'
    )
    l10n_it_amount_withholding_signed = fields.Monetary(
        string='Withholding',
        compute='_compute_amount_extended',
        currency_field='company_currency_id'
    )
    l10n_it_amount_before_withholding_signed = fields.Monetary(
        string='Total Before Withholding',
        compute='_compute_amount_extended',
        currency_field='company_currency_id'
    )
```